python/google/protobuf/internal/type_checkers.py

```diff
@@ -45,7 +45,7 @@
 
 __author__ = 'robinson@google.com (Will Robinson)'
 
-import ctypes
+import struct
```

Comment:
Use the original try import struct https://github.com/protocolbuffers/protobuf/commit/75eff10d81233d3d08ce5ded215e526c1fcace88

And add comment that this is for cpython 3.11's new wasm target

Reply:
@anandolee Updated! It seems this fork has not been approved to run tests, though. Is there anything else I need to do to get approval. thanks again!